warp10/src/main/java/io/warp10/script/aggregator/And.java

```diff
@@ -60,8 +64,11 @@ public Object apply(Object[] args) throws WarpScriptException {
         continue;
       } else {
         and = and && Boolean.TRUE.equals(values[i]);
-        if (!and) {
-          break;
+        
+        if (ticks[i] > timestamp) {
```

Comment:
What happens if the GTS has duplicate timestamps? What happen if one timestamp is MIN_VALUE?